src/lib/containers/drug_upload_tool/DrugUploadTool.tsx

```diff
@@ -108,13 +108,22 @@ class DrugUploadTool extends React.Component<
           token,
         )
         formData = JSON.parse(fileData)
+      } else {
+        //if we are creating a new file - store the versions
+        formData = {
+          metadata: {
```

Comment:
Essential in the future (when the schema definitions inevitably change), very nice.